### DISCUSSION TITLE
Can current Trilium display in a window smaller than full screen?

### DISCUSSION THREAD

**Discussion by LorenAmelang** [TriliumNext/Trilium#4477] [Q&A] [answered]
On Windows 10 I used to keep Trilium in a window sized smaller than my full screen, so I could peek at bits of data in other apps. I don't know for sure when things changed, maybe when I switched to Arch Linux and Cinnamon...  Or maybe using VNC to access it changed something, but it acts the same when accessed from the local machine. 

Now whenever I start Trilium, or expand it up from the system tray/panel, it fills the whole screen. Covers the tray/panel icons. Using F11 or the square icon that I assume is window size, I can show or hide the tray/panel. But I can't find a way to shrink the Trilium window any smaller. No drag icons along any sides or corners. 

Options -> Appearance -> Zoom Factor will shrink the content dramatically, but the window boundaries stay locked to my full screen, or the portion over the tray/panel. 

Am I missing something?

**Comment by meichthys** [comment]
This is definitely not my experience on Windows. Perhaps you could try removing the data folder in your appdata directory and re-downloading. (Just be sure you have a backup in case something goes wrong)

**Comment by zadam** [comment]
Yes, Trilium works in non-full screen mode. It sounds like a problem with your Window Manager (Cinnamon in your case).

> No drag icons along any sides or corners.

This is controlled by the Window Manager.

**Reply by LorenAmelang** [reply, marked as answer]
Did some searching, found "Window Tiling" - "Enable window tiling and snapping" was on. Turned it off, restarted Trilium, it loaded above the tray/panel icons. Tried the top right square icon, it shrank to a nice smaller window with draggable corners. Seems to work as I'd expect now - Thanks! 

I hit "snapping" far more often by accident than intention anyway - glad it is gone. Maybe that's how Trilium got maximized originally...

**Reply by LorenAmelang** [reply]
Somehow this stopped working on 0.63.5, or maybe on some update of Cinnamon. Every time I move to another app window and back to Trilium it covers all my Linux tray icons. The Maximize square can move it off the icons, but it still occupies the rest of the full screen. Right click on the Trilium tray icon Unmaximize option shrinks it, but nothing can change the default size that sets.  

I found a "Windows" panel in the App menu Settings (separate from the Preferences -> System Settings -> Window Tiling option), that has a "Moving and Resizing Windows" section with a "Special Key" option. Setting that to Ctrl lets me use Ctrl+left click anywhere in the Unmaximized Trilium window to drag it around. Still can't resize it, even with the border width for resize instead of move set to max. (But can't resize any other app that way...)

Inconvenient...  All my other apps get the normal edge arrows for resizing.

**Reply by LorenAmelang** [reply]
Wow! I just used the Web Clipper extension (that works well many times a day) to try to save this page to Trilium. Got a repeating mess:

```
We don’t support that file type.

Try again with GIF, JPEG, JPG, MOV, MP4, PNG, SVG, WEBM, CPUPROFILE, CSV, DMP, DOCX, FODG, FODP, FODS, FODT, GZ, JSON, JSONC, LOG, MD, ODF, ODG, ODP, ODS, ODT, PATCH, PDF, PPTX, TGZ, TXT, XLS, XLSX or ZIP.

Attaching documents requires write permission to this repository.

Try again with GIF, JPEG, JPG, MOV, MP4, PNG, SVG, WEBM, CPUPROFILE, CSV, DMP, DOCX, FODG, FODP, FODS, FODT, GZ, JSON, JSONC, LOG, MD, ODF, ODG, ODP, ODS, ODT, PATCH, PDF, PPTX, TGZ, TXT, XLS, XLSX or ZIP.

This file is empty.

Try again with a file that’s not empty.

This file is hidden.

Try again with another file.

Something went really wrong, and we can’t process that file.

Try again.
```


A second try included my post content among the error messages, in the same note as the first mess. ???
